lib/presentation/mypage/my_page.dart

```diff
@@ -79,8 +82,28 @@ class MyPage extends ConsumerWidget {
                         ),
                         ListTileWithIcon(
                           title: myPageItemi18n.aboutThisApp,
-                          onTap: () async =>
-                              const AboutAppPageRouteData().push<void>(context),
+                          onTap: () async {
+                            final packageInfo =
+                                await PackageInfo.fromPlatform();
+                            if (context.mounted) {
+                              unawaited(
+                                Navigator.push(
+                                  context,
+                                  MaterialPageRoute(
+                                    builder: (context) => Theme(
+                                      data: Theme.of(context).copyWith(
+                                        cardColor: Colors.white,
+                                      ),
+                                      child: LicensePage(
+                                        applicationName: packageInfo.appName,
+                                        applicationVersion: packageInfo.version,
```

Comment:
buildNumberも含めて表示して欲しいです！

```suggestion
applicationVersion: '${packageInfo.version}+${packageInfo.buildNumber}',
```